seed_data/management/commands/seed_db.py

```diff
@@ -200,6 +202,7 @@ def deserialize_user_data_list(user_data_list, course_runs):
     for user_data in user_data_list:
         new_user = deserialize_user_data(user_data, course_runs)
         new_user_count += 1
+        Profile.objects.create(user=new_user)
         deserialize_model_data_on_object(new_user.profile, user_data)
```

Comment:
Nitpick: it would be clearer to save the result of `Profile.objects.create(user=new_user)` into a variable called `new_profile`, and refer to that on the subsequent lines, rather than referring to `new_user.profile` on those lines.